src/style.css

```
@@ -1 +1,8 @@
-/* styles go here */
+.label-margin {
+  margin-bottom: 10px;
+  display: block;
+}
+
+.group {
+  margin-bottom: 20px !important;
```

```suggestion
  margin-bottom: 20px;
```
Never ever use important! rule